### DISCUSSION TITLE
Error: No www/ directory found!

### DISCUSSION THREAD

**Discussion by JAS-75** [html2app/discussions#7] [General]
Hi, I allwas get the same error "No www/ directory found!" but in the .zip file is the directory www.

Can you take a look to my .zip file and tell me were is the problem?

[my-app.zip](https://github.com/html2app/discussions/files/10993371/my-app.zip)

**Comment by yandeu** [comment]
Strange. Maybe it has something to do with the huge size (11.8 MB) of your index.html file?

**Reply by yandeu** [reply]
I just uploaded it and it compiled successfully.
